components/Service/Snuboard/Snuboard.tsx

```diff
@@ -0,0 +1,58 @@
+import classNames from "classnames/bind";
+import useWaffleScroll from "../../../library/waffleScroll";
+import IntroductionHead from "../common/IntroductionHead/IntroductionHead";
+import IntroductionCarousel from "../common/IntroductionCarousel/IntroductionCarousel";
+import styles from "./Snuboard.module.scss";
+
+const cx = classNames.bind(styles);
+
+function Snuboard() {
+  const { ref, scrollState } = useWaffleScroll(
+    ({ toggleState }) => {
+      toggleState(0.75, 2.7, "available");
+    },
+    { available: false },
+  );
+
+  return (
+    <section className={cx("container", scrollState)} ref={ref}>
+      <div className={cx("background")}>
+        <div className={cx("bars")}>
+          {/* 아래 Array<number>(8)에서 number은 any를 쓰지 않기 위한 임의 지정 type */}
+          {[...Array<any>(8)].map((_, index) => (
+            <div key={index} className={cx("bar", `bar${index + 1}`)}></div>
+          ))}
```

Comment:
```suggestion
          {Array.from(Array(8).keys()).map((index) => (
            <div key={index} className={cx("bar", `bar${index + 1}`)}></div>
          ))}
```
요런 식으로 빈 숫자 배열 구현하는 방법 도 있습니다! 불필요한 타입 지정이 사라진다는 장점이 있지만 배열을 두 번 생성하니 속도가 미묘하게 느려질 수도... 뭐 취향 차이인 듯 합니다

Reply:
위에 방법도 스프레더 쓰니까 배열을 두 번 생성한다고 봐야하나? 여튼 성능은 잘 모르겠네요

Reply:
이런 경우 저는 그냥 하드코딩하는 편입니다. 그게 싫으면 함수 하나 만드는게 제일 깔끔하겠네요. 아무튼 저런 이상한 코드는 지양하는 편...

Reply:
오오 감사합니다. 개인적으로는 하드코딩이 가장 깔끔할 거 같아서 하드코딩으로 반영했습니다.